hmpps-kotlin-spring-boot-autoconfigure/src/main/kotlin/uk/gov/justice/hmpps/kotlin/auth/AuthAwareTokenConverter.kt

```diff
@@ -59,8 +59,6 @@ open class AuthAwareAuthenticationToken(
   authorities: Collection<GrantedAuthority> = emptyList(),
 ) : JwtAuthenticationToken(jwt, authorities) {
   override fun getPrincipal(): String = userName ?: clientId
-
-  fun isSystemClientCredentials(): Boolean = userName == null
```

Comment:
This is the function being removed - fortunately it's not being used outside of the library so we can get away with a patch version bump.